processing/src/main/java/io/druid/query/groupby/GroupByQueryHelper.java

```diff
@@ -243,12 +243,10 @@ public Row apply(Row input)
 
     for (AggregatorFactory aggregatorFactory : query.getAggregatorSpecs()) {
       final String typeName = aggregatorFactory.getTypeName();
-      final ValueType valueType;
-      if (typeName != null) {
-        valueType = Enums.getIfPresent(ValueType.class, StringUtils.toUpperCase(typeName)).orNull();
-      } else {
-        valueType = null;
-      }
+      final ValueType valueType = GuavaUtils.getEnumIfPresent(
+          ValueType.class,
+          typeName == null ? null : StringUtils.toUpperCase(typeName)
```

Comment:
Suggested to make `GuavaUtils.getEnumIfPresent()` to not accept `null` value, and here, move the check out of `GuavaUtils.getEnumIfPresent()` call.